plugins/PostProcessingPlugin/scripts/MultiExtColorMix.py

```diff
@@ -0,0 +1,623 @@
+#        Without a 'Purge' the tool changes this Post applies are done 'On-the-Fly' and printing simply continues after any tool change.  If you select to insert a Purge, the tool head will move to the park position and purge itself and then continue.  This may leave a tit on a print.  If at the end of a Mix you want to switch to a single extrudere you should probably add a purge or it may be a bit before the color straightens out.
+#     If you end the mix at a mid-print layer; you can continue with the final mix or switch to another extruder.
+#   The sum of the Start Percent for the extruders enabled in this post must equal 100.
+#   The sum of the End Percent (when Gradient is used) for the extruders enabled in this post must also equal 100.
+#   Ex: If you are using two extruders in the mix and one starts at 66 then the other must start at 34.
+#   Ex: If you are using three extruders and the first starts with 35, and 2nd with 10 then E3 must start with 55.
+#   The same thing applies to the End numbers when you use a gradient mix.
+#   If the Percentage Sums do not add up then the post will exit with a message.
+#     The virtual extruder is always Extruder_count+1 (either T2 or T3 in the gcode)
+#     This Post can also be used to simply set an extruder for a print.
+#   May be used with either Absolute or Relative extrusion.
+#   If 'One-at-a-Time' printing and only Extruder #1 is enabled, ALL extruders will be available post-process.
+
+from ..Script import Script
+from UM.Application import Application
+from UM.Message import Message
+from cura.CuraApplication import CuraApplication
+import re
+
+class MultiExtColorMix(Script):
+    def __init__(self):
+        super().__init__()
+    # Enable the third extruder setting if there is one---------------------------------
+    def initialize(self) -> None:
+        super().initialize()
+            
+        ext_cnt = Application.getInstance().getGlobalContainerStack().getProperty("machine_extruder_count", "value")
+        if ext_cnt > 2:
+            enable_invis = True
+        else:
+            enable_invis = False
+        self._instance.setProperty("invis_setting", "value", enable_invis)
+    # Post a message if only a single instance of this post is running-------------------------------------------
+        multi_present = 0
+        pp_name_list = Application.getInstance().getGlobalContainerStack().getMetaDataEntry("post_processing_scripts")
+        for pp_name in pp_name_list.split("\n"):
+            if "MultiExtColorMix" in pp_name:
+                multi_present += 1
+        if multi_present == 1:
+            Message(text = "Multi-Extruder Color Mixer:" + "\n" + "You must have a Blending Hot End.  Extruders Share Heater' and 'Extruders Share Nozzle' must be enabled in the 'Printer Settings' in Cura.  Open 'MultiExtColorMix.py' in a text editor and review the opening lines for more instructions." + "\n").show()
```

Comment:
I think this message is a bit technical for the average cura user. We cannot expect users to open source code and read technical documentation.
```suggestion
            Message(text = "Multi-Extruder Color Mixer:" + "\n" + "You must have a Blending Hot End.  Extruders Share Heater' and 'Extruders Share Nozzle' must be enabled in the 'Printer Settings' in Cura.).show()
```

Reply:
Additionally such a message might be a bit verbose if a user is already aware of this requirement. Moving such a message in a less intrusive place might be better.

Reply:
Removed.
I have a pull request in for a post processor that simply opens a PDF document with instructions for all the post processors.  Aldo Hoeben commented on the fact that it "doesn't do anything" and it is true that it has no function as a post processor but it is documentation on the post processor settings that is right there for a user.  We'll see how that goes.